.github/workflows/e2e-cross-version.yml

```diff
@@ -1,8 +1,8 @@
 name: E2E Cross-version Tests
 
-on:
-  schedule:
-    - cron: '0 9 * * 0'
+on: [push]
+#  schedule:
+#    - cron: '0 9 * * 0'
```

Comment:
We can make this on demand with `workflow_dispatch`.

Reply:
Just for more clarity, I mean this as for after this PR. It's perfectly valid to have it run on every push while you're still iterating on it.

Reply:
It's back to the original version.